lib/util/cli.js

```diff
@@ -393,10 +393,6 @@ const Cli = {
         responses.client_id = responses.client_id.trim();
         responses.client_secret = responses.client_secret.trim();
         responses.auth_url = responses.auth_url.trim();
-        responses.account_id = responses.account_id.trim();
```

Comment:
first rename, the fully removing EID?
why? so far that acted as central identifier and second thing we test for to validate credentials are a match between auth and config file

Reply:
its not removing it, it just stops trimming because it is a number, so trim doesnt work :)